mozilla_bitbar_devicepool/lambdatest/user_script/setup_script.sh

```diff
@@ -0,0 +1,189 @@
+# mozilla_platform_ops_git_commit=b13c723154cebebc48e71566eede9b5129b675ea # right before jdk 17 upgrade
+# # https://raw.githubusercontent.com/mozilla-platform-ops/mozilla-bitbar-docker/b13c723154cebebc48e71566eede9b5129b675ea/taskcluster/worker-runner-config.yml.template
+# wget https://raw.githubusercontent.com/mozilla-platform-ops/mozilla-bitbar-docker/${mozilla_platform_ops_git_commit}/taskcluster/worker-runner-config.yml.template
+# wget https://raw.githubusercontent.com/mozilla-platform-ops/mozilla-bitbar-docker/${mozilla_platform_ops_git_commit}/scripts/entrypoint.sh
+# wget https://raw.githubusercontent.com/mozilla-platform-ops/mozilla-bitbar-docker/${mozilla_platform_ops_git_commit}/scripts/entrypoint.py
+# wget https://raw.githubusercontent.com/mozilla-platform-ops/mozilla-bitbar-docker/${mozilla_platform_ops_git_commit}/scripts/run_gw.py
+# wget https://raw.githubusercontent.com/mozilla-platform-ops/mozilla-bitbar-docker/${mozilla_platform_ops_git_commit}/taskcluster/script.py
+
+# # edit paths in entrypoint.*, run_gw.py, and maybe others /builds/taskcluster & /usr/local/bin -> /home/ltuser/taskcluster
+# sed -i 's/builds\/taskcluster/home\/ltuser\/taskcluster/g' entrypoint.sh
+# sed -i 's/builds\/taskcluster/home\/ltuser\/taskcluster/g' entrypoint.py
+# sed -i 's/builds\/taskcluster/home\/ltuser\/taskcluster/g' run_gw.py
+# sed -i 's/builds\/taskcluster/home\/ltuser\/taskcluster/g' worker-runner-config.yml.template
+# sed -i 's/usr\/local\/bin/home\/ltuser\/taskcluster/g' worker-runner-config.yml.template
+# sed -i 's/builds\/worker/home\/ltuser\/taskcluster/g' entrypoint.sh
+# sed -i 's/chown/# chown/g' entrypoint.sh # avoid chown worker
+
+# # hack to get android_serial to be added to the environment - important as we have usb+tcp listed with `adb devices`
+# sed -i 's/"TC_WORKER_GROUP",/"TC_WORKER_GROUP","ANDROID_SERIAL","UserPorts",/g' entrypoint.py
+
+# # hack on script.py - ideally fix this in the script.py itself
+# sed -i 's/builds\/worker/home\/ltuser\/taskcluster/g' script.py
+# sed -i 's/builds\/taskcluster/home\/ltuser\/taskcluster/g' script.py
+
+# # ignore error about >1 device, we have adb + tcp_ip connections for our current device.
+# sed -i s/sys.exit\(exit_code\)/#pass/g script.py
+
+# # need to figure out how to set this - scripts depend on this file existing
+# echo '80.0.0' > /home/ltuser/taskcluster/version
+
+# # we want entrypoint.sh to setup everything in the "pre-step", but the scenario needs to run "run_gw.py"
+# sed -i 's/run_gw.py/# run_gw.py/g' entrypoint.sh
+# # aje: busted, getting `sed: -e expression #1, char 13: unterminated `s' command`
+# # sed -i s/run_gw.py/# run_gw.py/g entrypoint.sh
+
+# # adjust defaults for taskIdleTimeout and cleanuptasksdir
+# #sed -i 's/5400/5/g' worker-runner-config.yml.template  # I assume we can use an exit code
+# echo "disableReboots:   true" >> worker-runner-config.yml.template  # this ensures that the docker container can be managed on it's own
+
```

Comment:
I can't wait for this to work and we can trust it enough to delete all of these comments :)

Reply:
I've stored the full files with your modifications in this PR/repo, so we can remove them... but they are helpful to explain what/why we've changed them.